packages/mobile/src/inHouseLiquidity/index.test.ts

```diff
@@ -0,0 +1,48 @@
+import { createPersonaAccount } from 'src/inHouseLiquidity'
```

Comment:
let's use kebab-case for the IHL directory. So, `packages/mobile/src/in-house-liquidity` instead of `packages/mobile/src/inHouseLiquidity`